pkgs/development/compilers/llvm/15/libcxx/default.nix

```diff
@@ -47,6 +47,15 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./gnu-install-dirs.patch
+    # See:
+    #   - https://reviews.llvm.org/D133566
+    #   - https://github.com/NixOS/nixpkgs/issues/214524#issuecomment-1429146432
+    # !!! Drop in LLVM 16+
+    (fetchpatch {
```

Comment:
I've split this to https://github.com/NixOS/nixpkgs/pull/253936 as it's ready while the other issue still needs reviewing